modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/util/ConsistencyCheckUtils.java

```diff
@@ -240,7 +240,7 @@ public static KeyCacheObject unmarshalKey(KeyCacheObject unmarshalKey,
                     cacheObjOpt.isPresent() ?
                         new PartitionReconciliationValueMeta(
                             cacheObjOpt.get().valueBytes(ctx),
-                            Optional.ofNullable(cacheObjOpt.get().value(ctx, false)).map(Object::toString).orElse(null),
+                            cacheObjOpt.map(o -> objectStringView(ctx, o)).orElse(null),
```

Comment:
For simple keys that'll return almost useless string representation: "CacheObjectImpl [val=null, hasValBytes=true]"
Optional.ofNullable(cacheObjOpt.get().value(ctx, false)).map(Object::toString) will return "abc_100".
As a result, in case of simple keys, partition reconciliation report in --verbose mode won't contain human readable version of per-node-values. Seems that GridCommandHandlerPartitionReconciliationAbstractTest'tests will fail.

Reply:
Fixed